include/z64scene.h

```diff
@@ -33,10 +33,10 @@ typedef struct {
     /* 0x0E */ s16   params;
 } TransitionActorEntry; // size = 0x10
 
-typedef struct TransitionActorContext {
-    /* 0x00 */ u8 numActors;
+typedef struct TransitionActorList {
+    /* 0x00 */ u8 num;
```

Comment:
I think MM's `count` is better than `num` here (of course it's a number, it's a u8). Ditto for RoomList

Reply:
Im sure num was used because a debug string says thats what its original name was-- but agreed, it should be count.
fwiw `num` is meant to mean `number of`, not just indicating that its a number